website_docs/instrumentation.md

```diff
@@ -17,30 +17,52 @@ ctx, span = tracer.Start(ctx, "helloWorld")
 defer span.End()
 ```
 
-In Go, the `context` package is used to store the active span. When you start a span, you'll get a handle on not only the span that's created, but the modified context that contains it. When starting a new span using this context, a parent-child relationship will automatically be created between the two spans, as seen here:
+In Go, the `context` package is used to store the active span. When you start a span, you'll get a handle on not only the span that's created, but the modified context that contains it.
+
+Once a span has completed, it is immutable and can no longer be modified.
+
+### Get the current span
+
+To get the current span, you'll need to pull it out of the `Context` you have a handle on:
+
+```go
+ctx := context.Background()
```

Comment:
Including this in the example might be misleading. This will mean the extracted span is always empty, invalid, and non-recording. It would likely make sense to explain in a comment about how `ctx` needs to come from some parent process that has already set an active span instead of this line.

Reply:
Yeah, I was wondering about that. The previous example did this, but my understanding is that context is usually sources from a request object or something like that - More than happy to adjust and explain here, including rework the sample.